stable/filebeat/templates/daemonset.yaml

```diff
@@ -31,8 +31,8 @@ spec:
 {{- if .Values.priorityClassName }}
       priorityClassName: "{{ .Values.priorityClassName }}"
 {{- end }}
-{{- if .Values.indexTemplateLoad }}
       initContainers:
+{{- if .Values.indexTemplateLoad }}
```

Comment:
will this fail if neither indexTemplateLoad nor extraInitContainers are used?

Reply:
This was able to start without a declaration of either values

```
$ helm --name example-filebeat install stable/filebeat/
$ helm list
NAME                    REVISION        UPDATED                         STATUS          CHART                   NAMESPACE
example-filebeat        1               Tue Aug 28 14:06:33 2018        DEPLOYED        filebeat-0.7.1          defaulr
$ kubectl --namespace=default get pods -l "app=filebeat,release=example-filebeat"
NAME                     READY     STATUS    RESTARTS   AGE
example-filebeat-69t97   1/1       Running   0          1m
```

Reply:
you can have an empty initContainer section.